wallet-gateway/remote/src/user-api/server.ts

```diff
@@ -23,14 +24,16 @@ export const user = (
     userUrl: string,
     notificationService: NotificationService,
     drivers: Partial<Record<SigningProvider, SigningDriverInterface>>,
-    store: Store & AuthAware<Store>
+    store: Store & AuthAware<Store>,
+    networkCacheStore: NetworkCacheStore
```

Comment:
I think calling this a `networkCacheStore` might be slightly misleading since it doesnt implement the Store interface. How about just `NetworkCache`?